src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -1385,6 +1385,11 @@ void MarkEntireAssembly (AssemblyDefinition assembly)
 				MarkingHelpers.MarkForwardedScope (new TypeReference (exportedType.Namespace, exportedType.Name, module, exportedType.Scope));
 			}
 
+			// Mark scopes of type references by traversing the assembly.
+			new TypeReferenceMarker (assembly, MarkingHelpers).Process ();
```

Comment:
Which type references we are converting are missing in module.GetTypeReferences () ?

Reply:
`GetTypeReferences` misses ones like in the original issue, for example when a field type is a builtin type. There's no typeref row for the reference in metadata in that case.